platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -191,6 +191,22 @@
 	 */
 	public static native void method(String p_sname, String p_name, String p_ret, String[] p_params);
 
+	/**
+	 * Used to complete registration of the {@link org.godotengine.godot.Godot.SingletonBase} instance's methods.
+	 * @param p_sname Name of the instance
+	 * @param p_name Name of the signal to register
+	 * @param p_params Signal parameters types
+	 */
+	public static native void addsignal(String p_sname, String p_name, String[] p_params);
```

Comment:
Here and below, can you update the method name to comply with java naming style (e.g: `addSignal`).

Reply:
done